numpy/linalg/linalg.py

```diff
@@ -858,6 +858,27 @@ def qr(a, mode='reduced'):
 
     a, wrap = _makearray(a)
     _assertRank2(a)
+    m, n = a.shape
+    if _isEmpty2d(a):
+        k = min(m, n)
+        if mode == 'reduced':
+            # ‘reduced’ : returns q, r with dimensions (M, K), (K, N) (default)
```

Comment:
The forward quotes are unicode, causing a test failure

Reply:
oh... actually intended to remove those lines; leaving them there and fixing the quotes.